src/apps/popup/pages/receive/content.tsx

```diff
@@ -43,7 +43,7 @@ export const ReceivePageContent = () => {
           <Trans t={t}>Receive tokens</Trans>
         </Typography>
       </ParagraphContainer>
-      <ActiveAccountPlate label="To account" />
+      <ActiveAccountPlate label="To account" symbol={'CSPR'} balance={null} />
```

Comment:
Why you didn't pass the balance prop here?

Reply:
This is not related to this work